panther_analysis_tool/main.py

```diff
@@ -230,6 +233,11 @@ def load_analysis_specs(directories: List[str]) -> Iterator[Tuple[str, str, Any,
                 # skip loading files that have already been imported
                 if spec_filename in loaded_specs:
                     continue
+                # Dont load files that are explictly ignored
+                relative_name = os.path.join(relative_path, filename)
```

Comment:
So if I remember correctly about what `os.walk` returns this will yield a path like `./test.yml`. The comparison will fail if the user has entered `test.yml`, right? Maybe you want to pass both paths through [`os.path.normpath`](https://docs.python.org/3/library/os.path.html#os.path.normpath) and then compare?